src/pages/docs/feature-toggles/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-05-21
+modDate: 2025-05-21
+title: Feature Toggles 
+navTitle: Feature Toggles
+navSection: Feature Toggles
+description: Octopus Feature Toggles allow progressive delivery of changes and instant rollback 
+navOrder: 95 
+---
+
+Octopus Feature Toggles support toggling features on or off in real-time, without redeploying, and progressively releasing changes to subsets of your users.
+
+:::div{.hint}
+Octopus Feature Toggles are currently in Alpha, available to a small set of customers. 
+
+If you are interested in this feature please register your interest on the [roadmap card](https://roadmap.octopus.com/c/121-feature-toggles) and we'll keep you updated.
+:::
+
+## Usage 
+
+### Create a Feature Toggle 
+
+Feature Toggles are located within Octopus Projects: 
+**Project ➜ Feature Toggles**
+
+Create a new Toggle and give it name.
+
+![New toggle name](/docs/feature-toggles/new-toggle-name.png "width=500")
+
+### Configure OpenFeature in your client application {#configure-open-feature-client-app}
+
+Octopus Feature Toggles rely on [OpenFeature](https://openfeature.dev/) as the client SDK.
+
+Follow the [OpenFeature guide for installing the SDK for your language](https://openfeature.dev/ecosystem?instant_search%5BrefinementList%5D%5Btype%5D%5B0%5D=SDK) into your application. 
+
+Configure OpenFeature to use the [Octopus Provider](#providers).
+
+The Octopus OpenFeature Provider requires a client identifier when instantiated. This is a [JWT](https://jwt.io/introduction) which specifies the Octopus Project, Environment, and Tenant (if applicable). This tells the Octopus Feature Toggle service which set of toggles to evaluate.
+
+:::div{.hint}
+The Octopus Feature Toggle client identifier is available via the Octopus variable `Octopus.FeatureToggles.ClientIdentifier` or via the Feature Toggle UI (see below). 
+:::
+
+For applications deployed by Octopus, the recommended way is to have Octopus inject the client identifier as part of deployment, for example by injecting it into a configuration file or environment variable. The client identifier is made available via the Octopus variable `Octopus.FeatureToggles.ClientIdentifier`.   
+
+For applications not deployed by Octopus, or cannot have the client identifier supplied during deployment for any reason, the client identifier can be obtained via the portal UI, as shown below.
+
+![Client identifier preview menu item](/docs/feature-toggles/client-identifier-preview-menu-item.png "width=500")
```

Comment:
Optional! Astro doesn't recognize these widths, so they can be safely removed.

```suggestion
![Client identifier preview menu item](/docs/feature-toggles/client-identifier-preview-menu-item.png)
```

Reply:
I've removed all these widths in a single commit. Ta!